packages/calypso-products/src/translations.tsx

```diff
@@ -1248,75 +1252,75 @@ export const getJetpackProductsWhatIsIncluded = (): Record< string, Array< Trans
 
 	return {
 		[ PRODUCT_JETPACK_AI_MONTHLY ]: [
-			translate( '100 monthly requests (upgradeable)' ),
+			highlightable( translate( '100 monthly requests (upgradeable)' ) ),
```

Comment:
Instead of declaring those wrappers here, I'd recommend doing it in the component that needs them. These constants can be used in a bunch of other places and we can't easily test them all, so we can't easily ensure that adding the wrappers doesn't break something. 

Also, this list of constants is big already, so adding additional components and markup here is unreasonable and undesired IMHO.

Reply:
The reason I didn't do this from the beginning was that the info shows on the lightboxes is pretty much entirely defined here in this file. The best way I can think to do this is to add this wrapper to the first item in the includes list where it is used if the product is multi-tiered 🤔  But that seems fragile and I tend to like declarative solutions more. 

Thinking through this more... A solution where the list iteslf can tell which items have been changed and when they were changed would probably be better so we wouldn't have to deal with multiple lines that can potentially be changed non-simultaneously later on. I think implementing a solution like that would solve most of the issues you brought up